src/api/app/models/token/workflow.rb

```diff
@@ -42,8 +42,8 @@ def call(options)
   end
 
   def owned_by?(some_user)
-    # TODO: remove the first condition if we migrate the Token.user to Token.users
-    user == some_user || users.include?(some_user) || groups.map(&:users).flatten.include?(some_user)
+    # TODO: remove the first condition if we migrate the Token.user to Token.users_shared_among
```

Comment:
I believe this comment should be updated:

```suggestion
    # TODO: remove the first condition if we migrate the Token.executor to Token.users_shared_among
```